### PR TITLE
Converts circuitboards to GAGS

### PR DESCRIPTION
**Pull request by Thunder12345**
## About The Pull Request

Converted computer/machine boards to use GAGS, tweaked a few of the colours to make them more distinct in the process.

unknown
New boards on top, old on the bottom

## Why It's Good For The Game

Converts another obvious candidate to use GAGS, replaces 8 icon states with 3.

## Changelog
:cl:
imageadd: The engineering circuitboards are now yellow instead of orange.
/:cl:

### CODE REVIEW

**Review by Twaticus — APPROVED** [review]

**Comment by maxymax13** [comment]
don't like how the yellow is so much less warm

**Comment by CRITAWAKETS** [comment]
These all look better aside from engiboards. They're supposed to be orange, not yellow.

**Comment by Thunder12345** [comment]
Twat thought orange looked too similar to supply brown, so it's yellow instead

**Comment by maxymax13** [comment]
coders when they see more than 3 colored sprites in a spritesheet (they get to make a pr where they suck out all the hue shifting for minimal sprite bloat loss)

**Comment by CRITAWAKETS** [comment]
> coders when they see more than 3 colored sprites in a spritesheet (they get to make a pr where they suck out all the hue shifting for minimal sprite bloat loss)

these sprites dont even have hue shifting, take your meds

**Comment by maxymax13** [comment]
> > coders when they see more than 3 colored sprites in a spritesheet (they get to make a pr where they suck out all the hue shifting for minimal sprite bloat loss)
> 
> these sprites dont even have hue shifting, take your meds

yes they do, not my fault you have cataracts

**Comment by Fikou** [comment]
> coders when they see more than 3 colored sprites in a spritesheet (they get to make a pr where they suck out all the hue shifting for minimal sprite bloat loss)

sneed

**Comment by bluetshirtguy** [comment]
When you thought getting more soulful wasn't possible.

**Comment by Mothblocks** [comment]
i think the bikeshed should be orange, with a darker yellow

**Comment by Mothblocks** [comment]
i updated your changelog to be player facing

File: code/__DEFINES/colors.dm [hunk 1]
```diff
@@ -169,3 +169,13 @@
...
+
+//The GAGS greyscale_colors for each department's computer/machine circuits
+#define CIRCUIT_COLOR_GENERIC list("#1A7A13")
+#define CIRCUIT_COLOR_COMMAND list("#1B4594")
+#define CIRCUIT_COLOR_SECURITY list("#9A151E")
+#define CIRCUIT_COLOR_SCIENCE list("#BC4A9B")
+#define CIRCUIT_COLOR_SERVICE list("#92DCBA")
+#define CIRCUIT_COLOR_MEDICAL list("#00CCFF")
+#define CIRCUIT_COLOR_ENGINEERING list("#F8D700")
+#define CIRCUIT_COLOR_SUPPLY list("#C47749")
```
Comment (Mothblocks) [inline code/__DEFINES/colors.dm:181]: I don't think these defines should be lists.

You should turn these into generic color defines (like the rest of this file), then just make a list everytime you call greyscale_colors.

In the way it is now, it's confusing--`greyscale_colors = CIRCUIT_COLOR_ENGINEERING` looks like `CIRCUIT_COLOR_*` is a normal color, not a list. You could change the variable names to match, as well, but I like the other approach better.

**Pull request merged**